src/corelibs/U2Gui/src/ObjectViewModel.h

```diff
@@ -245,6 +248,16 @@ protected slots:
     OptionsPanel *optionsPanel;
 };
 
+/** Constants for known GObject view menu types. */
+class U2GUI_EXPORT GObjectViewMenuType {
+public:
+    /** A constant for building a contextual (popup) menu. See buildMenu for details. */
+    const static QString CONTEXT;
```

Comment:
I think, it's better to use `enum`

Reply:
Enums are sealed (closed) types. Here we have an open set of values that can be extended by views. I described the way I plan extending it in MSA editor.